File: 28_annation_processing/src/main/java/com/hand/a28_annation_processing/MainActivity.java
package com.hand.a28_annation_processing;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {


    // 默认的value 方法 可以简写
//    @BindView(R.id.tv)
//    TextView mTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

//        Binding.bind(this);
//
//        mTextView.setText("one bit ");


    }
}

File: motionlayoutsample/src/main/java/com/kohler/simple/motionlayoutsample/MainActivity.kt
package com.kohler.simple.motionlayoutsample

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        btn_object.setOnClickListener {
            startActivity(Intent(this, ObjectActivity::class.java))
        }

        btn_object2.setOnClickListener {
            startActivity(Intent(this, Object2Activity::class.java))
        }

        btn_go.setOnClickListener {
            startActivity(Intent(this, GoActivity::class.java))

        }

        btn_go2.setOnClickListener {
            startActivity(Intent(this, Go2Activity::class.java))

        }

        btn_set.setOnClickListener {
        startActivity(Intent(this, ConstrainSetActivity::class.java))

        }
    }
}
